### PR TITLE
Replace max and min with defines

### PR DESCRIPTION
**Pull request by TheKnarf**

### CODE REVIEW

File: src/math/minmax.h [hunk 1]
```diff
@@ -1,29 +1,9 @@
...
-}
-
-inline size_t max_size(size_t a, size_t b)
-{
-    return a > b ? a : b;
-}
+#define min_int64(a, b) (a < b ? a : b)
+#define max_int64(a, b) (a > b ? a : b)
+#define min_size(a, b) (a < b ? a : b)
+#define max_size(a, b) (a > b ? a : b)
```
Comment (rexim) [inline src/math/minmax.h]: `a` and `b` are evaluated twice. Try

```c
min_int64(printf("hello\n"), printf("world\n"));
```

Comment (TheKnarf) [reply src/math/minmax.h]: Oh.

Comment (rexim) [reply src/math/minmax.h]: @TheKnarf welcome to the C preprocessor buddy! \o/

**Comment by TheKnarf** [comment]
@rexim Updated PR.

**Pull request merged**